app/controllers/products_controller.rb

```diff
@@ -1,12 +1,13 @@
 class ProductsController < ApplicationController
+  protect_from_forgery with: :null_session
   def show
     @product = Product.find(params[:id])
     render json: @product
   end
 
   def index
-    @product = Product.all
-    render json: @product
+    @products = Product.where('user_id = ' + current_user.id.to_s)
```

Comment:
```suggestion
    @products = Product.where(user_id: current_user.id)
```

It's good not to serialize/escape query arguments yourself and let ActiveRecord do this for you.